Examples/UseCases.ipynb

```diff
@@ -158,7 +182,11 @@
    "source": [
     "### Adding Annotations\n",
     "\n",
-    "Similar to [DKPro cassis](https://github.com/dkpro/dkpro-cassis), to add manual annotations to cas object, we need to defined it with `typesystem`. For the given text, annotations of Tokens that has an id and pos feature can be added in the following.\n"
+    "Similar to [DKPro cassis](https://github.com/dkpro/dkpro-cassis), to add manual annotations to cas object, we need to defined it with `typesystem`. For the given text, annotations of Tokens that has an id and pos feature can be added in the following.\n",
+    "\n",
+    "**REC**: The `CAS` type in Java UIMA is written with all upper-case letters. I think it would introduce an unnecessary difference to call it `Cas` here.\n",
+    "\n",
+    "**REC**: Why does `CAS(dts().typesystem)()` have two sets of parentheses? That looks very odd."
```

Comment:
Can be replaced the whole line with:

```dkpro_typesystem =  dts().typesystem``` or ```dkpro_typesystem =  dts(version="2.0.0").typesystem```
```Cas(dkpro_typesystem)()``` 

We need parenthesis again to initialize Cas object. 

Please provide your feedback on this.

Reply:
`dkpro_typesystem = dts(version="2.0.0").typesystem` looks better but is still strange. Maybe `dkpro_core_typessystem = DKProCore(version="2.0.0").get_typesystem()`?

I still don't get it. The CAS object can be constructed using `Cas(typesystem=DKProCore(version="2.0.0").get_typesystem())`, right? Why put another set of parentheses after it?

Reply:
I see your point, For now, I am using **pydkpro** Cas class as a wrapper on the top of **dkpro-cassis** Cas along with some helper functions. So, double parenthesis is used to call **pydkpro** Cas class method to constructs the **dkpro-cassis**  Cas object. I believe, in that way, we can avoid redundancy in the code. If you have any second opinion, please let me know. I should have mentioned about this in earlier comment.

Reply:
Would subclassing work? I.e. let the pydkpro CAS inherit from the cassis CAS and let the pydkpro CAS call the constructor from the superclass?

Reply:
yes, that is the correct way. I'll do the update. Thanks!!